docs/source/edge_networks.rst

```diff
@@ -0,0 +1,66 @@
+Support for Edge Networks
+=========================
+
+Recptor out-of-the-box has the ability to support complicated networking environments including edge networks.
+
+Consider the following environment:
+
+.. image:: edge.png
+
+Configurable-Items
+^^^^^^^^^^^^^^^^^^
+
+Receptor encapsulates the concepts of `below-the-mesh` and `above-the-mesh` connections. Please refer to :doc:`tls` for a better understanding of these networking layers.
+
+If a particular node in a network has higher than normal latency, we allow the users to define a finely-grained idle connection timeout value for any given Receptor node. This will help Receptor keep `below-the-mesh` tcp connections alive. Receptor will attempt to reconnect to a timed-out node, to a maximum retry limit and if this retry limit is surpassed, Receptor will drop the connection from the routing table.
```

Comment:
"to a maximum retry limit and if this retry limit is surpassed"

it's not so much retry logic and limits, rather just monitoring traffic that flows over a backend connection.

If the last statement in this paragraph should be something like,
```
Receptor will monitor backend connections for traffic and will timeout any connection that hasn't seen traffic for a period of time. Once the connection is dropped, a new connection is formed automatically. 
```

Reply:
verbatim copy/paste :) thanks for the suggesstion